dev-client/babel.config.js

```diff
@@ -2,7 +2,7 @@ module.exports = api => {
   api.cache(true);
 
   return {
-    presets: ['module:@react-native/babel-preset'],
+    presets: ['babel-preset-expo', '@babel/preset-typescript'],
```

Comment:
Question: Can you explain what this is doing?

Reply:
babel is a javascript transpiler (transforms javascript to more compatible/faster javascript). it is used as a pre-processing step before jest tests the code. here we're changing the preset configuration babel uses to presets that better support our codebase. i was confused why the project was working without having the expo/typescript preset, i think it might be because babel is _only_ used when running jest, but i'm not sure